feat(experiments): "Make decision" touch ups

jurajmajerik opened the pull request:
## Changes
Small tweaks based on @annikaschmid feedback:

<img width="436" alt="image" src="https://github.com/user-attachments/assets/821256df-ba76-4487-9fa1-beefcc3a354d">

- using IconRocket now. Another fun one could be IconConfetti. Available icons here: https://storybook.posthog.net/?path=/story/posthog-3000-icons--alphabetical
- added a button tooltip

## How did you test this code?
👀

jurajmajerik committed:
init

annikaschmid commented:
A ship as an icon would be cool. For ‘shipping’ 🚢

The other thing I am not sure on is the button colour. Do we use this anywhere else in the app in light mode? It's very present, which I like, but I am unsure whether it's consistent

annikaschmid commented:
Maybe the wording should be:
"Choose variant and roll out to all users"
OR
"Choose a variant and roll it out to all users"

So you either have those small words in both places, or none

jurajmajerik committed:
changes

jurajmajerik commented:
- discussed with Cory and changed the color
- changed the tooltip text

![image](https://github.com/user-attachments/assets/c21035d4-6278-4649-a298-048471abff96)

we don't have a ship icon 😞

jurajmajerik committed:
rework based on cory's feedback

github-actions[bot] committed:
Update UI snapshots for `chromium` (1)

github-actions[bot] committed:
Update UI snapshots for `chromium` (2)

jurajmajerik committed:
resolve

jurajmajerik committed:
Merge branch 'make-decision-ui-tweaks' of https://github.com/PostHog/posthog into make-decision-ui-tweaks
